addons/mail/static/src/components/message/message.xml

```diff
@@ -120,6 +124,7 @@
                                 </t>
                             </div>
                         </t>
+                        <MessageInReplyToView t-if="messageView.messageInReplyToView and threadView and threadView.threadViewer.chatWindow" messageInReplyToViewLocalId="messageView.messageInReplyToView.localId"/>
```

Comment:
It's wrong to show the item at the bottom of message. It should always be shown at the very top of the message, including in chat window